### ISSUE TITLE
Thread starting during runtime shutdown

### ISSUE THREAD

**Issue by shaheenkdr** [tolmachevroman/AcraSlack#3]
When there is an exception, the app closes, I think OkHTTP is trying to send data at that time!. Anyway to force send the error only when the app is restarted back?

**Comment by tolmachevroman** [comment]
You're right, @shaheenkdr. Sometimes error report is not sent because the very reporting tool (ACRA and Retrofit) shut down with the app.
 _You could_ store the error in the SharedPreferences as a JSON string, for example. Then on app start check if the value is not empty and send the error message.

**Comment by shaheenkdr** [comment]
Thanks for replying! In which file should I make a change? It'd be great if you can make a change in master itself! Then I guess this is the perfect way to use ACRA!

**Comment by tolmachevroman** [comment]
@shaheenkdr Let me investigate more on ACRA, I'll keep you posted.

**Comment by shaheenkdr** [comment]
Glad to know!
